org.adempiere.base/src/org/compiere/model/MProcessDrillRule.java

```diff
@@ -154,7 +155,7 @@ public static MProcessDrillRule[] getByTable(Properties ctx, int AD_Table_ID, St
 	 */
 	public static MProcessDrillRule[] getByColumnName(Properties ctx, String columnName, String trxName) {
 
-		String whereClause = "";
+		String whereClause = " IsValid = 'Y' ";
```

Comment:
I see the getByTable is not used anywhere, but in case is used in future, must not have the same where filter for valid rules?

Reply:
Yes, thank you, it is fixed